extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -63,10 +63,14 @@ amp-autocomplete > input {
   border-radius: 0px 0px 4px 4px; /* default-ui-border-radius */
 }
 
-.i-amphtml-autocomplete-item:hover {
+.i-amphtml-autocomplete-item:hover:not([disabled]) {
   background-color: rgba(0,0,0,.15); /* default-ui-light-gray */
 }
 
-.i-amphtml-autocomplete-item-active {
+.i-amphtml-autocomplete-item-active:not([disabled]) {
   background-color: rgba(0,0,0,.15); /* default-ui-light-gray */
 }
+
+.i-amphtml-autocomplete-item[disabled] {
+  color: silver;
```

Comment:
/cc @andrewwatterson because you might have an opinion on what this color should be.

Reply:
I've struggled a lot with the Default UI color system, but for the time being let's go with `rgba(0,0,0,.33)`

Nota bene that disabled items don't need to adhere to the same contrast requirements per WCAG standards.